### ISSUE TITLE
Readme.md has incorrect example code.

### ISSUE THREAD

**Issue by ncadigan** [sal0max/grav-plugin-shortcode-gallery-plusplus#5]
[gallery
    rowHeight=230
    margins=25
    lastRow="justify"
    captions="false"
    border=0]
\![Alt text 1]\(image.jpg)
\![Alt text 2]\(/images/image.jpg)
\![relative link]\(../image.jpg)
\![remote link]\(https://remotesite.com/image.jpg)
...
[/gallery]
**Above code does not work. It displays as plain text.**

**Keeping it as one line does work though.**
[gallery rowHeight=230 margins=25 lastRow="justify" captions="false" border=0]
\![Alt text 1]\(image.jpg)
\![Alt text 2]\(/images/image.jpg)
\![relative link]\(../image.jpg)
\![remote link]\(https://remotesite.com/image.jpg)
...
[/gallery]

**Comment by sal0max** [comment]
Thanks for the report.

I tested my example again, and it works. However, trying different configurations, I found a way that breaks it.
What's your setting in `/user/config/system.yaml` for the key `pages|markdown|auto_line_breaks`? `true` breaks the page, `false` renders properly. `false` is also the default value.

So, this is basically an issue only in some user configurations.

But to not confuse anyone, I will probably change the example.

Cheers!
